python/whylogs/core/preprocessing.py

```diff
@@ -199,6 +197,54 @@ def _pandas_split(self, series: pd.Series, parse_numeric_string: bool = False) -
         self.bool_count = bool_count
         self.bool_count_where_true = bool_count_where_true
 
+    def _polars_split(self, series: pl.Series, parse_numeric_string: bool = False) -> None:
+        """
+        Split a Polars Series into numpy array and other Polars series.
+
+        Args:
+            series: the original Pandas series
+            parse_numeric_string: if set, this will coerce values into integer using pands.to_numeric() method.
+
+        Returns:
+            SplitSeries with multiple values, including numpy arrays for numbers, and strings as a Polars Series.
+        """
+
+        # TODO: add a PolarsView, or convert PandasView to work with both Polars & Pandas
```

Comment:
We should extract a `View` base class and make `ListView, NumpyView, PandasView, PolarsView <: View`. `PolarsView` might be better than `NumpyView` for ints and floats. `PreprocessedColumn` would contain a list of `View` instances rather than members of each specific `View` subclass.